### ISSUE TITLE
Issues with ember-cli-htmlbars-inline-precompile 1.0

### ISSUE THREAD

**Issue by kellyselden** [kategengler/ember-cli-code-coverage#133]
Context https://github.com/ember-cli/ember-cli-htmlbars-inline-precompile/issues/90#issuecomment-321696056

```
Error: Plugin 1 specified in "base" provided an invalid property of "_parallelBabel"
    at Plugin.init
```

**Comment by rwwagner90** [comment]
@kellyselden is this something we need to fix here or in ember-cli-htmlbars-inline-precompile?

**Comment by kellyselden** [comment]
Here I believe https://github.com/ember-cli/ember-cli-htmlbars-inline-precompile/issues/90#issuecomment-321696056

**Comment by rwwagner90** [comment]
@kellyselden ah I see. @rwjblue how do we ensure transpilation does not occur here?

**Comment by rwjblue** [comment]
Will be fixed by https://github.com/kategengler/ember-cli-code-coverage/pull/141.

**Comment by theseyi** [comment]
Wondering if there's a workaround for this issue in the interim @kategengler ?

**Comment by rwwagner90** [comment]
@theseyi I don't think there is a current workaround, but definitely going to try to prioritize the rewrite. Could probably use some help, if you are interested!

**Comment by theseyi** [comment]
@rwwagner90 sure willing to help where I can. Is there any fundamental issue with progressing with #141 ?

**Comment by rwwagner90** [comment]
@theseyi it's just not done. It's an initial spike. We need to make sure we get it to the same level of functionality currently provided.

**Comment by rwwagner90** [comment]
This should be fixed now, please try out 1.0.0-beta.3. If there are any remaining issues, please let us know!
